bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/internal/ImageList.java

```diff
@@ -22,17 +22,19 @@ public class ImageList {
 	long handle;
 	int style, refCount;
 	Image [] images;
+	final int zoom;
 
-public ImageList (int style) {
-	this (style, 32, 32);
+public ImageList (int style, int currentZoomLevel) {
```

Comment:
Rename the parameter to zoom as well for consistency. See the other constructor as well

Reply:
Done